gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidHandler.java

```diff
@@ -199,4 +232,29 @@ private String redFlagReasonError(boolean isCgmFileAvailable, boolean isCracFile
         return stringJoiner.toString();
     }
 
+    private void runDichotomyForFullImport(TTimestamp timestamp, CseValidRequest cseValidRequest, TcDocumentTypeWriter tcDocumentTypeWriter) {
+        DichotomyResult<RaoResponse> dichotomyResult = dichotomyRunner.runDichotomy(cseValidRequest, timestamp);
+        if (dichotomyResult != null && dichotomyResult.hasValidStep()) {
+            TLimitingElement tLimitingElement = this.limitingElementService.getLimitingElement(dichotomyResult.getHighestValidStep());
+            BigDecimal mibniiValue = timestamp.getMiBNII().getV().subtract(timestamp.getANTCFinal().getV());
+            BigDecimal mniiValue = computeMnii(dichotomyResult).map(Math::round).map(BigDecimal::valueOf).orElse(mibniiValue);
+            tcDocumentTypeWriter.fillTimestampWithDichotomyResponse(timestamp, mibniiValue, mniiValue, tLimitingElement);
+        } else {
+            tcDocumentTypeWriter.fillDichotomyError(timestamp);
```

Comment:
You can call fillTimestampError with proper message directly

Reply:
No, the error statusNumber differs in the case of dichotomy error and the case of other errors : it is one for dichotomy but zero for other cases. Should I add the statusNumber as parameter in the fillTimestampError method? Not sure if that is better though.